### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by bahorn**

### CODE REVIEW

**Comment by bahorn** [comment]
To resolve #198

File: .github/ISSUE_TEMPLATE/new-event.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+labels: New event
+assignees: ''
+
+---
+
+## Event Name
+
+## Date
+
+* DATE HERE
```
Comment (robzwolf) [inline .github/ISSUE_TEMPLATE/new-event.md:14]: Date format would be useful

File: .github/ISSUE_TEMPLATE/new-event.md [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+
+## Is the even primarily for students?
```
Comment (robzwolf) [inline .github/ISSUE_TEMPLATE/new-event.md:16]: Typo – should be `event`

**Comment by bahorn** [comment]
@tomMisson mentioned that they we're looking a implementing a nicer way of taking even submissions than PRs earlier, so I'd wait for them to clarify.

**Comment by tomMisson** [comment]
I'll have a look in two ways to handle phone data and see if there's an easy way or not so difficult way to implement forms so we could have a structure on the website to take information. Probably won't be till next weekend though now so feel free to review this if you think it's ready @bahorn ?

File: .github/ISSUE_TEMPLATE/new-event.md (hunk 1 above)
Comment (bahorn) [reply .github/ISSUE_TEMPLATE/new-event.md]: Went for `YEAR-MONTH-DAY` because I love ISO dates, though probably a nicer way to write it out.

Comment (robzwolf) [reply .github/ISSUE_TEMPLATE/new-event.md]: I mean, ISO-8601 is objectively the best date format 😎

**Comment by bahorn** [comment]
Yeah this is fine for review now

**Pull request merged**